### ISSUE TITLE
Color contrast does not meet minimum requirement - v2 pager

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9640]
# Motivation

Siteimprove is reporting a color contrast issue with a pager on a SiteNow v2 site.

<img width="1469" height="812" alt="Image" src="https://github.com/user-attachments/assets/a596fb45-447a-4e38-becf-5025cb648b4f" />

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r69

Siteimprove link to existing issue - https://my2.siteimprove.com/Inspector/1007696/A11Y/Page?pageId=47297576796&impmd=6F4B1BAAA780A4D7A3CA00D93433A366&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r69/failed/VEdKbjRMSUU=;ZWMxZjc0OTE=

Examples
- https://businsvc.fo.uiowa.edu/

**Color contrast does not meet minimum requirement**
Issue
1.4.3: Contrast (Minimum)
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.
